include/kyosu/types/impl/quaternion/axes.hpp

```diff
@@ -0,0 +1,24 @@
+//======================================================================================================================
+/*
+  Kyosu - Complex Without Complexes
+  Copyright : KYOSU Contributors & Maintainers
+  SPDX-License-Identifier: BSL-1.0
+*/
+//======================================================================================================================
+#pragma once
+
+namespace kyosu
+{
+  template < int N > struct axes:std::integral_constant<int, N>{};
+  inline constexpr auto _X(axes<1>{});
+  inline constexpr auto _Y(axes<2>{});
+  inline constexpr auto _Z(axes<3>{});
+
+  template < bool e > struct ext:std::integral_constant<bool, e>{};
+  auto Extrinsic(ext<true>{});
+  auto Intrinsic(ext<false>{});
+
+  template < bool e > struct nor:std::integral_constant<bool, e>{};
+  inline constexpr auto Assume_normalized(nor<false>{});
```

Comment:
same, why the capitalization ?

Reply:
It was my choice, that can change. But mainly because it was nor a decorator nor a variable